src/components/Main/Main.tsx

```diff
@@ -0,0 +1,33 @@
+/** @jsxImportSource @emotion/react */
+import { useLocation } from "react-router-dom";
+
+import { MainStyle } from "./Main.style";
+
+import CartItemContainer from "./Cart/CartItemContainer/CartItemContainer";
+import CartResults from "./Cart/CartResults/CartResults";
+import CartTitle from "./Cart/CartTitle/CartTitle";
+
+import OrderContainer from "./Order/OrderContainer";
+
+const Main = () => {
+  const { pathname } = useLocation();
+
+  if (pathname !== "/" && pathname !== "/order") {
+    return;
+  }
+
+  const routeTitle = {
+    "/": (
+      <>
+        <CartTitle></CartTitle>
+        <CartItemContainer></CartItemContainer>
+        <CartResults></CartResults>
```

Comment:
```suggestion
        <CartTitle />
        <CartItemContainer />
        <CartResults />
```

로 줄여도 되겠죠?

더불어서 이 제목 객체의 위치가 Main 컴포넌트가 적절한지에 대해서도 조금 더 고민해볼까요?